client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -267,16 +266,24 @@ class EmailPlan extends React.Component {
 	}
 
 	renderAddNewMailboxesOrRenewNavItem() {
-		const { domain, hasSubscription, purchase, selectedSite, translate } = this.props;
+		const { domain, hasSubscription, purchase, translate } = this.props;
 
-		if ( ! domain.currentUserCanManage || ! hasSubscription ) {
-			return null;
+		if ( hasEmailForwards( domain ) ) {
+			return (
+				<VerticalNavItem { ...this.getAddMailboxProps() }>
+					{ translate( 'Add new email forwards' ) }
+				</VerticalNavItem>
+			);
 		}
 
-		if ( ! selectedSite || ! purchase ) {
+		if ( ! purchase ) {
```

Comment:
It looks like this change I've recommend in https://github.com/Automattic/wp-calypso/pull/53189/files#r643073055 wasn't applied in the end.

Reply:
I remember applying that change. I must have missed pushing that up.